src/components/time-selector/TimeSelector.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react'
+import { checkboxArray } from './Checkboxes'
+import { ICheckbox } from './TimeSelectorInterfaces'
+import { TimeSelectorCard } from '../card/TimeSelectorCard'
+
+export const TimeSelector = () => {
+
+    const [cards, setCards] = useState<string[]>([])
+    const [checkboxes, setCheckboxes] = useState<ICheckbox[]>(checkboxArray)
+
+    const getCards = () => {
+        return ( 
+            cards?.map(name => {
+            return name && <TimeSelectorCard key={name} name={name}/>
```

Comment:
`<TimeSelectorCard key={`${index}-${card.name}`} {...card}/>`

Reply:
cards => array of strings. we cannot do this. we have to pas prop as name={card or cardName}